exporter/clickhouselogsexporter/config.go

```diff
@@ -41,7 +45,10 @@ type Config struct {
 	UseNewSchema        bool   `mapstructure:"use_new_schema"`
 	LogLevelConcurrency *int   `mapstructure:"log_level_concurrency"`
 
-	AttributesLimits AttributesLimits `mapstructure:"attributes_limits"`
+	AttributesLimits          AttributesLimits `mapstructure:"attributes_limits"`
+	PromotedPathsSyncInterval *time.Duration   `mapstructure:"promoted_paths_sync_interval"`
+	ActivateBodyJSONCols      bool             `mapstructure:"activate_body_json_cols"`
+	CleanStringBasedBody      bool             `mapstructure:"clean_string_based_body"`
```

Comment:
these names can be better 

`body_json_enabled` -> default false 
`body_json_old_body_enabled` -> defult false which means if the body json is enabled by default it will clean the old body